build/doc_sync.py

```
@@ -0,0 +1,68 @@
+                skill_path = os.path.join(folder_path, filename, 'SKILL.md')
+                if os.path.exists(skill_path):
+                    title, desc = parse_aff(skill_path)
+                    output.append(f"| [`{filename}/`](./{folder}/{filename}/SKILL.md) | {desc} |")
+                continue
+                
+            title, desc = parse_aff(os.path.join(folder_path, filename))
+            output.append(f"| [`{filename}`](./{folder}/{filename}) | {desc} |")
+        
+        output.append("\n")
+    
+    output.append("---\n*Generated automatically by `/doc-sync` on " + os.popen('date').read().strip() + "*")
```

Ruff (0.15.15)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)